product_docs/docs/pgd/3.7/harp/01_release_notes/harp2.3.2_rel_notes.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: "Version 2.3.2"
+---
+
+This is a patch release of HARP 2 that includes fixes for issues identified
+in previous versions.
+
+| Type | Description |
+| ---- |------------ |
+| Bug fix | Multiple fixes related to network partition in BDR DCS. |
```

Comment:
Use &nbsp; to stop the Bug fix text being broken over lines:

```suggestion
| Bug&nbsp;fix | Multiple fixes related to network partition in BDR DCS. |
```

Reply:
Fixed